kafkabp/consumer.go

```diff
@@ -33,6 +33,11 @@ var (
 		Name:      "rebalance_total",
 		Help:      "The number of times consumer rebalance happened",
 	}, rebalanceLabels)
+
+	// Register the error counter so that it can be monitored.
+	_ = rebalanceCounter.With(prometheus.Labels{
+		successLabel: prometheusbp.BoolString(false),
+	})
```

Comment:
I think I would prefer this to be in an `init()`.

I think you should also register both `true` and `false`, no?

Reply:
we can register both true and false, it's just register true will not be very useful. `reset` is called in the constructor, so as soon as someone construct a consumer the true counter should emit 1.